workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md

```diff
@@ -0,0 +1,527 @@
+
+**N.B.:** If you are _not_ using a Free Trial subscription, try setting `--node-count` to **3** instead.
+
+```bash
+az aks create --name shfhubcluster \
+    --resource-group shf_test_hub \
+    --ssh-key-value ssh-key-shfhubcluster.pub \
+    --node-count 1 \
+    --node-vm-size Standard_D2s_v3 \
+    --output table
+```
+* `--name` is the cluster name we defined in [Step 4: Choose a Cluster Name](#aks-step4).
+* `--resource-group` is the resource group we created in [Step 3: Create a Resource Group](#aks-step3).
+* `--ssh-key-value` is the ssh public key we created in [Step 5: Create an SSH key](#aks-step5).
+* `--node-count` is the number of desired nodes in the Kubernetes cluster.
+* `--node-vm-size` is the size of the nodes you wish to use, which varies based on the use-case of the cluster and how much RAM/CPU each user will need.
+
+**N.B.:** The default version of Kubernetes will be installed, you can use the `--kubernetes-version` flag to install a different version.
+
+**This step may take a few minutes to execute.**
+
+### 7. Get credentials from Azure for `kubectl` <a name="aks-step7"></a>
+
+This step automatically updates your Kubernetes client configuration file to be configured with the cluster we've just deployed.
+
+```bash
+az aks get-credentials --name shfhubcluster \
+    --resource-group shf_test_hub \
+    --output table
+```
+* `--name` is the cluster name defined in [Step 4: Choose a Cluster Name](#aks-step4).
+* `--resource-group` is the resource group created in [Step 3: Create a Resource Group](#aks-step3).
+
+### 8. Check the Cluster is Fully Functional <a name="aks-step8"></a>
+
+```bash
+kubectl get node
+```
+
+The output of this command should list one node (unless you changed `--node-count` in [Step 6: Create an Azure Container Service (AKS) Cluster](#aks-step6)) with a `STATUS` of `READY`.
+The `VERSION` field reports which version of Kubernetes is installed.
+
+Example output:
+```bash
+NAME                       STATUS   ROLES   AGE   VERSION
+aks-nodepool1-97000712-0   Ready    agent   19m   v1.9.11
+```
+
+## Setting up Helm <a name="helm"></a>
+
+Adapted from [Zero-to-JupyterHub: Setting up and Securing Helm](https://zero-to-jupyterhub.readthedocs.io/en/latest/setup-helm.html).
+
+Helm is the package manager for Kubernetes and is used for: installing, upgrading and managing applications on a Kubernetes cluster.
+Helm packages are called _charts_.
+
+Helm has two parts: a client (`helm`) and a server (`tiller`).
+Tiller runs inside your Kubernetes cluster as a pod in the `kube-system` namespace.
+Tiller manages _releases_ (installations) and _revisions_ (versions) of charts deployed on the cluster.
+When you run a `helm` command, the local Helm client sends instructions to `tiller` in the cluster which in turn makes the requested changes.
+
+> **Did you know?:** Kubernetes is Greek for "captain" or "helmsman". Get ready for the nautical theme!
```

Comment:
Should this be before you introduce helm and tiller? Or "In case you haven't noticed the nautical theme!".

Reply:
Haha, yeah it was just a bit of fun 😉